python/composio/client/collections.py

```diff
@@ -983,7 +983,7 @@ def get(  # type: ignore
             queries["integrationIds"] = ",".join(integration_ids)
         if len(trigger_names) > 0:
             queries["triggerNames"] = to_trigger_names(trigger_names)
-        return self._raise_if_empty(super().get(queries=queries))
+        return super().get(queries=queries)
```

Comment:
This is a breaking change that affects error handling across the codebase. Several components still rely on `NoItemsFound` exception:
1. CLI triggers.py
2. toolset.py for integration lookups
3. client initialization in __init__.py

Consider:
1. Adding a deprecation cycle
2. Updating dependent code
3. Documenting the change in release notes